.changeset/grumpy-squids-laugh.md

```diff
@@ -0,0 +1,19 @@
+---
+"thirdweb": minor
+---
+
+Update SIWE interface
+```ts
+import { inAppWallet, createWallet } from "thirdweb/wallets"
+import { mainnet } from "thirdweb/chains"
+
+const rabby = createWallet("io.rabby");
+const wallet = inAppWallet();
+
+const account = await wallet.connect({
+    client: MY_CLIENT,
+    wallet: rabby,
```

Comment:
I would still add strategy: "wallet" 

Seems more consistent with the rest no?

Reply:
Gah yes that's required I just keep forgetting